web/src/pm_list_data.ts

```diff
@@ -154,6 +163,10 @@ export function get_list_info(
             return true;
         }
 
+        if (conversation.is_deactivated) {
```

Comment:
We want to show the conversation in zoomed in view.

Reply:
Sorry - I think I'm missing something. It looks to me like on line 152 we check if `zoomed` is true, and if so we just immediately return ALL conversations. So in the zoomed in view, we would already be showing the deactivated conversations. Please let me know if I'm missing something.